{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -9,13 +7,13 @@ PROJECT_NAME = {{ cookiecutter.repo_name }}
 PYTHON_VERSION = {{ cookiecutter.python_version_number }}
 PYTHON_INTERPRETER = python
 
-
 #################################################################################
 # COMMANDS                                                                      #
 #################################################################################
 
 {% if cookiecutter.dependency_file != 'none' %}
 ## Install Python Dependencies
+.PHONY: requirements
 requirements:
```

Comment:
This feels like a clearer way of indicating what is a phony recipe and making sure you don't miss any.